pkg/api/conversion_test.go

```diff
@@ -22,6 +22,26 @@ import (
 
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/api/testapi"
+	"k8s.io/kubernetes/pkg/apis/extensions"
+	"k8s.io/kubernetes/pkg/labels"
+	"k8s.io/kubernetes/pkg/registry/controller"
+	"k8s.io/kubernetes/pkg/registry/daemonset"
+	"k8s.io/kubernetes/pkg/registry/deployment"
+	"k8s.io/kubernetes/pkg/registry/horizontalpodautoscaler"
+	"k8s.io/kubernetes/pkg/registry/ingress"
+	"k8s.io/kubernetes/pkg/registry/job"
+	"k8s.io/kubernetes/pkg/registry/limitrange"
+	"k8s.io/kubernetes/pkg/registry/namespace"
+	"k8s.io/kubernetes/pkg/registry/node"
+	"k8s.io/kubernetes/pkg/registry/persistentvolume"
+	"k8s.io/kubernetes/pkg/registry/persistentvolumeclaim"
+	"k8s.io/kubernetes/pkg/registry/pod"
+	"k8s.io/kubernetes/pkg/registry/podtemplate"
+	"k8s.io/kubernetes/pkg/registry/resourcequota"
+	"k8s.io/kubernetes/pkg/registry/secret"
+	"k8s.io/kubernetes/pkg/registry/serviceaccount"
+	"k8s.io/kubernetes/pkg/registry/thirdpartyresource"
+	"k8s.io/kubernetes/pkg/registry/thirdpartyresourcedata"
```

Comment:
Good point. I exported `TestSelectableFieldLabelConversionsOfKind` with common functionality in `pkg/api/testing` and made it be invoked from respective directories of each resource.